docs/en/latest/FAQ.md

```diff
@@ -64,3 +64,15 @@ In such a case, you may need to apply these CRDs by yourself.
 ```shell
 kubectl apply -k samples/deploy/crd/
 ```
+
+### 8. Why apisix-ingress-controller reports similar errors when running: no matches for kind "ApisixRoute" in version "apisix.apache.org/v2beta1"?
+
+The version `v2beta1` is the newest version of resource `ApisixRoute`. You need to check if the CRD definition file is up to date.
+
+```shell
+kubectl get crd -A
```

Comment:
```suggestion
kubectl get crd apisixroutes.apisix.apache.org -o jsonpath='{ .spec.versions[*].name }' -A
```

Reply:
It would be better to get the results. like this:

```
kubectl get crd apisixroutes.apisix.apache.org -o jsonpath='{ .spec.versions[*].name }' -A
v1 v2alpha1 v2beta1                      
```

Reply:
nice